enhancements/etcd/protecting-etcd-quorum-during-control-plane-scaling.md

```diff
@@ -295,8 +295,8 @@ operator.
    (eg `my-cluster-master-3`)
 1. Take a copy of the Machine resource: `oc get machine -n openshift-machine-api my-cluster-master-0 -o yaml > my-
    cluster-master-3.yaml`
-1. Modify the Machine YAML to update the `name` and the size of the instance (eg. changing
-   `spec.providerSpec.value.instanceType` from `c5.xlarge` to `c5.2xlarge`) and remove `providerID`.
+1. Modify the Machine YAML to update the `metadata.name` field and the size of the instance (eg. changing
+   `spec.providerSpec.value.instanceType` from `c5.xlarge` to `c5.2xlarge`) and remove `spec.providerID` field and remove the entire status section and remove the `metadata.annotations` and `metadata.generation` fields and remove the `metadata.resourceVersion` and `metadata.uid` fields.
```

Comment:
Perhaps rather than one big long sentence, this would be better worded as
```suggestion
1. Modify the Machine YAML to update the `metadata.name` field and the size of the instance (eg. changing
   `spec.providerSpec.value.instanceType` from `c5.xlarge` to `c5.2xlarge`). The following fields will also need to be removed; `spec.providerID`, the entire `status` section, `metadata.annotations`, `metadata.generation`, `metadata.resourceVersion` and `metadata.uid`.
```

IIUC though, generation, resourceversoin and UID are set by the API server, so it doesn't matter if these are left in.

Why do we remove the annotations, can you provide and example of an annotation that causes an issue?

Reply:
I tried keep the annotations there, and it seems that new machine created successfully, so I think it could be ignore.

Reply:
/cc @JoelSpeed

Reply:
@geliu2016  , thanks for bringing it up , when I tried , I removed only providerID and the master machine was provisioned successfully seems to me that removing auto generated values is not needed. IMHO , the more values to remove the more confusion for the user.

Reply:
ok, I'm following OCP dr doc about deleting machine steps:
https://docs.openshift.com/container-platform/4.10/backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.html

so I don't know whether there are some potential risk if miss any steps or perhaps the document itself need to be updated.

Reply:
Thanks @geliu2016 , it does make sense then . 
LGTM .